### DISCUSSION TITLE
what about the "PageRank" meaning?

### DISCUSSION THREAD

**Discussion by YeChengMinMin** [st3v3nmw/obsidian-spaced-repetition#430] [Q&A] [answered]
is PageRank a yaml value，I need write in the mardown file，like this？
![image](https://user-images.githubusercontent.com/59163865/160750395-6a1b83e0-5e87-4e65-ae3e-5123e2d7069c.png)
I want the #review notes can show me in a special sequence---the high priority show me first，but i don't know how to set **pagerank** value

**Comment by dev-msp** [comment, marked as answer]
Page rank isn't anything you can set manually, it's an [algorithm](https://en.wikipedia.org/wiki/PageRank) that assigns scores to nodes in a graph (e.g. your notes) based on how many other nodes link to them.

I don't think there's any ability at the moment to specify the order of reviewable notes besides random or according to page rank.
